style.css

```diff
@@ -18,16 +21,17 @@ body {
 }
 
 .app-title{
-    width: 300px;
+    font-weight: bold;
+    width: 500px;
     height: 50px;
     border-radius: 10px 10px 0 0;
 }
 
 .app-title p {
     text-align: center;
-    padding: 15px;
+    padding: 30px;
     margin: 0 auto;
-    font-size: 2.2em;
+    font-size: 2.5em;
```

Comment:
I'm rusty on my css but is rem better than em? You'd have to look it up.

Reply:
yes rem is better, rem relates back to the root HTML font size vs em relates back to the parent element's font size